### DISCUSSION TITLE
Does this software support lrf files?

### DISCUSSION THREAD

**Discussion by arilando** [fraxiinus/ReplayBook#295] [Feature Request]
I became aware of this software because it allows replaying old LoL replays. Does it support lrf files? I have a bunch of old replays from 2014-2016 in that file format.

EDIT: The answer appears to be no, indeed it cannot detect older LoL clients (2016 or older) either.

**Comment by fraxiinus** [comment]
I believe my old application was able to view the metadata for LRF files: https://github.com/fraxiinus/ROFL-Player

Unfortunately it doesn't support playing the replays, only displaying metadata.

**Reply by JerryLiew** [reply]
Is it feasible to use the command line to replay these old replay file with the corresponding old client?

**Reply by fraxiinus** [reply]
maybe? i havent tried before... since these systems were before the introduction of rofl files, i doubt that the client would have a command that could play from a file. 

i assume that apps like LolReplay hosted a local spectate server much like how op.gg batch files work. so its definitely possible but i just havent dont any research in this area.
